testSrc/integration/io/flutter/integrationTest/NewProjectUITest.kt

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2025 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+package io.flutter.integrationTest
+
+import com.intellij.driver.sdk.ui.components.UiComponent.Companion.waitFound
+import com.intellij.driver.sdk.ui.components.common.ideFrame
+import com.intellij.driver.sdk.ui.components.common.toolwindows.projectView
+import com.intellij.driver.sdk.waitForIndicators
+import com.intellij.ide.starter.driver.engine.BackgroundRun
+import com.intellij.ide.starter.driver.engine.runIdeWithDriver
+import com.intellij.ide.starter.junit5.config.UseLatestDownloadedIdeBuild
+import io.flutter.integrationTest.utils.newProjectWelcomeScreen
+import org.junit.jupiter.api.*
+import org.junit.jupiter.api.extension.ExtendWith
+import java.nio.file.Paths
+import kotlin.time.Duration.Companion.minutes
+
+@Tag("ui")
+@TestInstance(TestInstance.Lifecycle.PER_METHOD)
+@ExtendWith(UseLatestDownloadedIdeBuild::class)
+class MyProjectUITest {
+
+  companion object {
+    // Generate a unique folder name for the test project to avoid conflicts
```

Comment:
This seems like an odd place for the comment. It feels like it should be in `initContext`. (Unless you want to say: "this will be updated to a unique folder name..."

Reply:
Moved

Reply:
Cool. Thanks!